react/features/base/media/components/web/Video.js

```diff
@@ -200,4 +207,22 @@ class Video extends Component<Props> {
     }
 }
 
-export default Video;
+/**
+ * Maps (parts of) the Redux state to the associated Video props.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     _noAutoPlayVideo: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    const testingConfig = state['features/base/config'].testing;
+
+    return {
+        _noAutoPlayVideo: testingConfig && testingConfig.noAutoPlayVideo
```

Comment:
You can use nullish coalescing here: `testingConfig?.noAutoPlayVideo`

Reply:
me likey, will do, thanks @saghul